eap74-rhel8-byos-multivm/src/main/scripts/create-ds.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+log() {
+    while IFS= read -r line; do
+        printf '%s %s\n' "$(date "+%Y-%m-%d %H:%M:%S")" "$line" >> /var/log/jbosseap.install.log
+    done
```

Comment:
https://stackoverflow.com/questions/26479562/what-does-ifs-do-in-this-bash-loop-cat-file-while-ifs-read-r-line-do

Slick.

Reply:
This reminds me that we have a lot of scripting surface area that might benefit from a comprehensive refactoring review. But that's for later.